robot_approach_distance: 1.05
arm_approach_distance: 0.05
forward_distance: 0.1
backward_distance: 0.2